src/schema/helpers.js

```diff
@@ -24,6 +37,9 @@ export const listToEnums = list => {
   if (!list) {
     return [];
   }
+  if (!Array.isArray(list)) {
+    list = objToArray(list);
```

Comment:
Could we use [`Object.values`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values) here?

```suggestion
    list = Object.values(list);
```

Reply:
Well shoot i didn't even remember this existed. Yes! Much simpler update :)